src/AutorestSwift/Code Generator/Manager.swift

```diff
@@ -216,4 +216,13 @@ class Manager {
             SharedLogger.fail("Fail to run tool \(tool).")
         }
     }
+
+    func saveCodeModel() {
+        guard let documentsUrl = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else {
```

Comment:
why we need to save the code model to yaml file? is that only for debugging? How will this file get removed?

Reply:
Yeah, it's just for debugging if you want to preserve the code model file when running autorest. It won't normally be invoked. I removed the line that calls this.